common/config.ts

```diff
@@ -78,6 +78,37 @@ export const UNIT_VISIBILITIES: ValueName[] = [
     { value: 'fremen', name: "Fremenská" }
 ]
 
+export const ALL_DECREE: string[] = [
+    "Mandát sardaukarů",
+    "Zrušení mandátu sardaukarů",
+    "Vyslání vojsk Landsraadu",
+    "Změna mandátu vojsk Landsraadu",
+    "Stažení vojsk Landsraadu",
+    "Zřízení vyšetřovací komise",
+    "Přerozdělení podílů CHOAM",
+    "Uvalení embarga",
+    "Zrušení embarga",
+    "Změna kvót",
+    "Jiný",
+    "- Žádný -"
+]
+
+export const SPECIAL_DECREE: string[] = [
```

Comment:
Tohle bude hlavne na ten specialni dekret (separatni PR) ale uz jsem to nechala v tomhle PR, snad to neva.

Reply:
I wouldn't do a array of special decrees, I imagine a UI where you need to enable checkbox for Bill of particulars. It's a rare event and it should not be distracting every round. This is quite complicating the code and it's possible that you won't have time to finish this feature until Monday. It's OK to send Bill of particulars via Slack. Definately remove it from this PR.

Reply:
Ok no probs.

Reply:
Removed.